src/main/java/com/flansmod/common/teams/CommandTeams.java

```diff
@@ -267,15 +267,34 @@ public void processCommand(ICommandSender sender, String[] split) {
             sender.addChatMessage(new ChatComponentText("Autobalance is now " + (TeamsManager.autoBalance ? "enabled" : "disabled")));
             return;
         }
-        if (split[0].equals("useRotation")) {
```

Comment:
It looks like you've removed the useRotation command, it needs to be added back

Reply:
sure

Reply:
This has been resolved.